plugins/linux-v4l2/v4l2-input.c

```diff
@@ -796,6 +797,9 @@ static obs_properties_t *v4l2_properties(void *vptr)
 	obs_properties_add_bool(props, "buffering",
 				obs_module_text("UseBuffering"));
 
+	obs_properties_add_bool(props, "close_on_hide",
```

Comment:
Use the string `deactivate_when_not_showing` to mirror windows dshow devices.

Reply:
@kkartaltepe Thank you. There is another issue and that is the spelling of variable names. A recently merged PR added two other boolean variables to the data struct and there CamelCase was used. Which I now have also used for my boolean. However all other variables in the struct have kebab_case. So shouldn't I use kebab_case too and the other two variables be changed (in a different patch/pr) also?